test/unit/bridge-client.test.js

```diff
@@ -289,10 +297,10 @@ describe('bridge-client', function () {
 
             // check they are deleted
             vm.runInContext(`
-                expect(typeof __uvm_dispatch).not.be('undefined');
-                expect(__uvm_dispatch).be(null);
-                expect(typeof __uvm_emit).not.be('undefined');
-                expect(__uvm_emit).be(null);
+                expect(typeof __uvm_dispatch).to.not.be.undefined;
```

Comment:
This assertion incorrectly checks for the type of the result of `typeof`, which will compare a string to `undefined`.